packages/error/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@frontity/error",
+  "version": "1.0.0",
```

Comment:
We have to start with a `0.0.1` or `0.1.0` version so when we do a `changeset version` with a major change for this package it will release the `1.0.0` version. If we start in `1.0.0` it releases the `2.0.0`.